tests/risk/data/all-keys-token.json

```diff
@@ -0,0 +1,60 @@
+{
+    "admin_key": {
+        "_type": "ED25519",
+        "key": "49efa5f54192470706914df50f17472f6f2310b8476293822e0d49b313ca2875"
```

Comment:
You may want to put the value of these keys into the repo's secrets so they're not viewable, but its test data so your call

Reply:
Thanks for the suggestion! It's testdata indeed, it's copied from the HASH CRABS GEN 2.0 mainchain token information: https://mainnet-public.mirrornode.hedera.com/api/v1/tokens/0.0.1270555/ So it's public data anyways.